cmd/keys.go

```diff
@@ -131,61 +132,111 @@ func keysRestoreCmd(a *appState) *cobra.Command {
 		Use:     "restore chain_name key_name mnemonic",
 		Aliases: []string{"r"},
 		Short:   "Restores a mnemonic to the keychain associated with a particular chain",
-		Args:    withUsage(cobra.ExactArgs(3)),
+		Args:    withUsage(cobra.RangeArgs(2, 3)),
 		Example: strings.TrimSpace(fmt.Sprintf(`
-$ %s keys restore ibc-0 testkey "[mnemonic-words]"
-$ %s k r cosmoshub faucet-key "[mnemonic-words]"`, appName, appName)),
+$ %s keys restore  ibc-0 testkey "[mnemonic-words]"
+$ %s k r "[mnemonic-words]" cosmoshub faucet-key
```

Comment:
since this example does not use the `--restore-all` flag the first argument should be the chain_name, second argument should be the key name, and the third argument should be the mnemonic.